README.md

```diff
@@ -89,6 +89,8 @@ Here is an example of a basic load balancer setup setup:
 
 ## Configuration
 
+Note: Please follow instructions to install the [Cert Manager]('https://github.com/jetstack/cert-manager/releases/download/v1.0.2/cert-manager.yaml') if you are using the ALB Ingress Controller.
+
```

Comment:
Add the `>` character before `Note:` to quote this item.

Reply:
Done